tools/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+set(EXAMPLE_SRCS topology.cc)
+if (PIKEOS)
```

Comment:
Do not build tool for PikeOS for now (not really usable as-is anyway on this embedded OS)